### PR TITLE
added reset ramp to CancelAll command

### PR DESCRIPTION
**Pull request by Levercpu**

### CODE REVIEW

File: src/main/java/frc/robot/commands/CancelAll.java [hunk 1]
```diff
@@ -26,6 +29,7 @@ public void initialize() {
     ramp.setRampPos(Constants.RAMP_POS_STOW);
     shooter.setShooterMotorLeftRPM(0);
     shooter.setShooterMotorRightRPM(0);
+    new RetractAmp(amp);
```
Comment (neo-Aditya) [inline src/main/java/frc/robot/commands/CancelAll.java]: That's not going to do what you think it'll do

File: src/main/java/frc/robot/commands/CancelAll.java [hunk 2]
```diff
@@ -6,14 +6,17 @@
...
 import frc.robot.constants.Constants;
 import frc.robot.subsystems.Ramp;
 import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.Amp;
 
 public class CancelAll extends Command {
   /** Creates a new CancelAll. */
   Ramp ramp;
   Shooter shooter;
+  Amp amp;
```
Comment (batchen1) [inline src/main/java/frc/robot/commands/CancelAll.java]: remove this

**Review by batchen1 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/sequences/CancelAllSequence.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import frc.robot.commands.amp.RetractAmp;
+import frc.robot.subsystems.Amp;
+import frc.robot.subsystems.Ramp;
+import frc.robot.subsystems.Shooter;
+
+public class CancelAllSequence extends SequentialCommandGroup {
+    public CancelAllSequence(Ramp ramp, Shooter shooter, Amp amp) {
+        addCommands(
+            new RetractAmp(amp),
+            new WaitCommand(0.5),
```
Comment (neo-Aditya) [inline src/main/java/frc/robot/commands/sequences/CancelAllSequence.java]: Do you want to put this in Constants?

**Review by neo-Aditya — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/sequences/CancelAllSequence.java (hunk 3 above)
Comment (bengold22) [inline src/main/java/frc/robot/commands/sequences/CancelAllSequence.java]: Remove this WaitCommand, do not leave it here and add the value to constants.

**Review by bengold22 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/sequences/CancelAllSequence.java
Comment (neo-Aditya) [inline src/main/java/frc/robot/commands/sequences/CancelAllSequence.java:1]: You may want to make it so that it'll stop the drivetrain aswell.

**Review by neo-Aditya — CHANGES_REQUESTED** [review]

**Review by batchen1 — APPROVED** [review]

**Review by batchen1 — APPROVED** [review]

**Pull request merged**